src/manifest.json

```diff
@@ -56,7 +58,8 @@
         "https://*.ses.nsw.gov.au/Messages/Create",
         "https://*.ses.nsw.gov.au/Messages/Create/"
       ],
-      "js": ["contentscripts/messages/create.js"]
+      "js": ["contentscripts/messages/create.js"],
+      "run_at" : "document_end"
```

Comment:
Right, fair enough then. The only reason I brought this up is that the less complicated shit we put in `manifest.json` the better for if we decide to bundle all the files together.